earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
@bradh Currently, the `src/installer` is also obsolete stuff, though it can be cleaned up and updated to use for generating rpm/deb packages.
Maybe move the `earth_enterprise/installer` to the `earth_enterprise/legacy_installer`, and copy scripts we need (scripts that are being used in Aaron-installer) into the `earth_enterprise/installer`?